assets/translations/en-us.yaml

```diff
@@ -5780,6 +5780,7 @@ workloadPorts:
   addPort: Add Port
   remove: Remove
   addHost: Add Host
+  warning: 'The service port name: "{ originalName }" is already in use, please enter a new name or delete the service.'
```

Comment:
Can we get rid of this translation string since we're not using it now? 
Otherwise this seems to work well; once this line is removed I think it's g2g